h/accounts/models.py

```diff
@@ -123,6 +123,17 @@ def subscriptions(self, value):
         else:
             self.status = (self.status or 0) & ~0b100
 
+    @property
+    def invited(self):
+        return bool((self.status or 0) & 0b100)
```

Comment:
Also maybe add a `# FIXME` or some other marker to remove this once we've been through the claim cycle? There won't be any more use for this code after we've sent out all these emails.